### PR TITLE
adding a jump-to-here script for lab 5

### PR DESCRIPTION
**Pull request by whitel**
I am not sure on the "names" yet. The shell script can be used in any dir. We could also just have one shell script which took a lab number as a param.

### CODE REVIEW

File: labs/lab5/jump-to-here-playbook.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - name: build wordpress
+    command: >
+      podman build 
+      -t wordpress 
+      -f /home/ec2-user/summit-2018-container-lab/labs/lab3/wordpress/Dockerfile.reference
+      /home/ec2-user/summit-2018-container-lab/labs/lab3/wordpress/
+    become: true
+
+  - name: launch registry
+    command: podman run --name registry -p 5000 -d registry
```
Comment (tchughesiv) [inline labs/lab5/jump-to-here-playbook.yml]: Once https://github.com/dustymabe/summit-2018-container-lab/pull/36 is merged, we have the students use a volume to persist the pushed images... not critical though.

File: labs/lab5/jump-to-here-playbook.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+--- 
+- hosts: localhost
+  become_method: sudo
+  tasks:
+  - name: build registry
+    command: podman build -t registry /home/ec2-user/summit-2018-container-lab/labs/lab1/registry
```
Comment (tchughesiv) [inline labs/lab5/jump-to-here-playbook.yml]: Once https://github.com/dustymabe/summit-2018-container-lab/pull/36 is merged... the registry directory will live under lab3.

**Comment by tchughesiv** [comment]
besides changing the registry build location to lab3... LGTM

File: labs/lab5/jump-to-here-playbook.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    command: podman push --tls-verify=false mariadb localhost:5000/mariadb
+    become: true
+
+  - name: load wordpress into registry
+    command: podman push --tls-verify=false wordpress localhost:5000/wordpress
+    become: true
+
+  - name: launch openshift
+    command: /bin/bash /home/ec2-user/start-oc.sh
+    become: false
```
Comment (whitel) [inline labs/lab5/jump-to-here-playbook.yml]: should add: ignore_errors: yes in case it is already running

**Comment by whitel** [comment]
I made some comments on the change to use persistent storage for the registry. Let's not approve/change/etc this one until we resolve that.

**Comment by tchughesiv** [comment]
responded to your comments...

File: labs/lab5/jump-to-here-playbook.yml (hunk 1 above)
Comment (tchughesiv) [reply labs/lab5/jump-to-here-playbook.yml]: disregard ... we've moved away from persistence for registry

File: labs/lab5/jump-to-here-playbook.yml (hunk 2 above)
Comment (scollier) [reply labs/lab5/jump-to-here-playbook.yml]: #36 has been merged.  We good here?

**Comment by scollier** [comment]
LGTM.  I didn't run them, but they seem reasonable.

File: labs/lab5/jump-to-here-playbook.yml (hunk 2 above)
Comment (tchughesiv) [reply labs/lab5/jump-to-here-playbook.yml]: still need to change to this to lab3:
```yaml
command: podman build -t registry /home/ec2-user/summit-2018-container-lab/labs/lab3/registry
```

**Pull request merged**